HW_7/Ex_34.py

```diff
@@ -0,0 +1,38 @@
+'''Задача 34:
+Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает, Вам стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг от друга пробелами. Стихотворение Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+Ввод:                                       Вывод:
+пара-ра-рам рам-пам-папам па-ра-па-дам      Парам пам-пам
+'''
+
+# ___________________РЕШЕНИЕ________________________
+
+
+def rythm_check(request):
+
+    request_list = list(map(str, request.split(' ')))
+
+    cnt1 = 0
+    cnt2 = list()
+
+    for phrase in request_list:
+        for letter in phrase:
+            for symbol in vowel_letters_ru:
+                if letter == symbol:
+                    cnt1 += 1
```

Comment:
Можно значительно проще, без последнего цикла. И множество можно задать тут явно. Переменная `vowel_letters_ru` ни к чему или сделайте эту сущность параметром
```suggestion
            if letter in {'а', 'у', 'о', 'ы', 'и', 'э', 'я', 'ю', 'ё', 'е'}:
                cnt1 += 1
```

Reply:
А еще множество использовать не обязательно. Можно просто одну строку.